invisible_cities/reco/corrections_new.py

```diff
@@ -229,11 +310,19 @@ def apply_all_correction_single_maps(map_e0     : ASectorMap,
         raise MissingArgumentError
         pass
 
+    if norm_strat is norm_strategy.max:
+        norm_value = amap_max(map_e0).e0
+    elif norm_strat is norm_strategy.mean:
+        norm_value = np.mean(np.mean(maps.e0))
```

Comment:
I dont see a reason to have separate dataclass for max values, while not for mean, so either we add amap_mean or  remove amap_max (and FitMapValue) (Im more for second option)